src/util/upnp_headers.cc

```diff
@@ -124,6 +124,6 @@ std::unique_ptr<std::map<std::string, std::string>> Headers::readHeaders(UpnpFil
         ret.insert(add);
     }
 
-    return std::make_unique<std::map<std::string, std::string>>(std::move(ret));
+    return ret;
 #endif
```

Comment:
We should keep the unique_ptr here to avoid copying the map.

Reply:
Removed.